prody/dynamics/plotting.py

```diff
@@ -336,6 +338,10 @@ def showProjection(ensemble, modes, *args, **kwargs):
 
     if SETTINGS['auto_show']:
         showFigure()
+
+    if returnAx:
```

Comment:
The axes is not returned because it can be simply retrieved via `matplotlib.pyplot.gca`. `show` is returned so that it is consistent with matplotlib.

Reply:
ok fine. I'll fix that then.

Reply:
Could you please tell me how to do that first.

Reply:
from matplotlib.pyplot import gca

ax = gca()

Reply:
ok great. That works so I'll revert that change.